### ISSUE TITLE
Minified martech plugin fails causes build to fail due to linting

### ISSUE THREAD

**Issue by johndavidsimmons** [adobe/aem-boilerplate#438]
### Expected Behaviour
Installing [AEM Martech Plugin](https://github.com/adobe-rnd/aem-martech) passes linting and builds successfully

### Actual Behaviour
Installing Martech plugin causes builds to fail due to linting errors. Build process is linting minified files.

### Reproduce Scenario (including but not limited to)
1. Make a new site using the boilerplate code.
2. Install the [AEM Martech plugin](https://github.com/adobe-rnd/aem-martech) 
3. Commit & Push to main triggering the build process
4. Build fails due to linting errors in minified martech plugin files

#### Sample Code that illustrates the problem
https://github.com/johndavidsimmons/aem-demo-site

**Comment by ramboz** [comment]
Since this is throwing errors on a minified file, I see 3 options here:
1. we explicitly call out in the martech lib documentation that you need to update your project's `.eslintignore`
2. we add a generic `*.min.js` in the boilerplate's `.eslintignore`
3. we consider it's the project team's responsibility to figure out what `.eslintignore` config to use based on the what they add to the boilerplate

I'm temped to do 1. and 3., but 2. would be a more generic future-proof approach… but this addresses something outside the scope of the boilerplate itself.

@kptdobe what's your take on this?

**Comment by kptdobe** [comment]
I agree option 2 would make sense: the chance a minified file passes the linting is really low...
